docs/trex_typescript.md

```diff
@@ -100,4 +100,4 @@ In the HTML code for the extensions, the JavaScript files are referenced. The fo
 
 - [TypeScript in Visual Studio Code](https://code.visualstudio.com/docs/languages/typescript?=target="_blank")
 
-- [TypeScript Sample Extensions (GitHub)](https://github.com/tableau/extensions-api/tree/master/Samples-TypeScript) and [Use the TypeScript samples]({{site.baseurl}}/docs/trex_examples.html#use-the-typescript-samples)
+- [TypeScript Sample Extensions (GitHub)](https://github.com/tableau/extensions-api/tree/main/Samples-Typescript) and [Use the TypeScript samples]({{site.baseurl}}/docs/trex_examples.html#use-the-typescript-samples)
```

Comment:
This last link `Use the TypeScript samples` isn't working for me.  I'm guessing maybe it's okay, and there's a preprocessor that runs and evaluates the `{{site.baseurl}}` clauses?

Reply:
Yep, the site.baseurl format there is part of the GitHub Pages / Jekyll system and will be replaced by the suitable hostname when published. However, that means they are not viewable in a PR or when looking directly at code in the repo. One way you can do it is to do a local docs build on this branch, and then you can see how the links are modified. We have a short summary of how to do this over in the [TSC project](https://tableau.github.io/server-client-python/docs/dev-guide#running-docs-locally).

A future conversation with Dave is whether to make these links consistent throughout the docs and if so, which convention to follow.